apidoc.raml

```diff
@@ -373,6 +373,7 @@ types:
       - new ideas: `?sort=-created`
       - ideas with provided creators: `?filter[creators]=username0,username1,username2`
       - ideas commented by provided users: `?filter[commentedBy]=username0,username1,username2`
+      - highly rated ideas with minimum rate parameter: `?filter[highlyRated]=bottomValueLimit`
```

Comment:
I don't understand the first question. Is it a separate question?
I don't understand the second question.

The naming part i separate from the rest?

Reply:
> I don't understand the first question. Is it a separate question?
I don't understand the second question.

Edited the comment to be clearer.

I want to understand a use case for different `bottomValueLimit`. I probably need an example when it would be useful to have the limit.
Why?
1. We will always get the highest voted ideas on page 1. It's up to frontend how far into the pages they want to go.
2. The limit different than 0 doesn't have a constant impact (i.e. at the beginning 5 can be high, but later (when everybody uses ditup and votes) 100 can be not enough)

> Is the naming part separate from the rest?

Yes.

Reply:
1) We would want to list 'the most popular' ideas which would be those with minimum 100 votes. I understand it would give us the similar result to 'get 20 first ideas most voted' but maybe we really want to list just 100+ ideas or ideas with positive votes or ideas which got more interest than ours.